cmd/kubeadm/app/util/pkiutil/pki_helpers.go

```diff
@@ -334,6 +332,18 @@ func TryLoadPrivatePublicKeyFromDisk(pkiPath, name string) (*rsa.PrivateKey, *rs
 	return k, p, nil
 }
 
+// TryLoadCSRFromDisk tries to load the CSR from the disk
+func TryLoadCSRFromDisk(pkiPath, name string) (*x509.CertificateRequest, error) {
+	csrPath := pathForCSR(pkiPath, name)
+
+	csr, err := CertificateRequestFromFile(csrPath)
+	if err != nil {
+		return nil, errors.Wrapf(err, "couldn't load the certificate request %s", csrPath)
+	}
+
+	return csr, nil
+}
```

Comment:
This function is just a wrapper around ` CertificateRequestFromFile` with added filename generation so I didn't add a unit test.
And it's only used in tests anyway.